Dockerfile.build

```diff
@@ -29,5 +29,5 @@ RUN apk add --no-cache ${ALPINE_PKG_BASE} ${ALPINE_PKG_EXTRA}
 WORKDIR /edgex-cli
 
 COPY go.mod .
-COPY ./res/configuration.toml /root/.edgex-cli/configuration.toml
+COPY ./res/.sample-configuration.toml /root/.edgex-cli/.sample-configuration.toml
```

Comment:
not fond of this sample file being hidden. It is clear enough that it is a sample by virtue of its name being  prefixed with sample.

Reply:
sample-configuration.toml file is not hidden anymore